nengo_viz/static/viz_netgraph.js

```diff
@@ -104,6 +131,7 @@ VIZ.NetGraph = function(parent, args) {
             self.offsetY = (self.offsetY + yy) / scale - yy;
             
             self.scale = scale * self.scale;
+            interact.margin(30*self.scale);
```

Comment:
I agree that it would be nice to have corner drag on ensembles, and it would be slightly different than the edge drag.  However, the current system does not allow that unless the ensemble happens to be about the right size.  If the Ensemble has been shrunk down (via edge drag) smaller than a certain size, then it is impossible to do anything other than a top-left corner drag, because the interact.margin is wider than the object itself.  If the Ensemble has been expanded (via edge drag) larger than a certain size, then you can't do any corner drags, and indeed even edge-drag becomes difficult because the interact.margin can become smaller than the distance from the visible circles to the edge of the invisible circle around the ensemble.  This is because the interact.margin is fixed to the global zoom level, rather than anything to do with the size of the object itself.  This makes the current scaled interact.margin system not work in lots of situations, and is why I think it needs to be removed before merging this PR.  But we should definite look into improving it in future PRs.  I just don't want this PR being blocked on this issue.  :)